docker-compose.dev.yaml

```
@@ -23,21 +82,50 @@ services:
       - POSTGRES_URL=${POSTGRES_URL}
       - POSTGRES_USERNAME=${POSTGRES_USERNAME}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
-    restart: always
     depends_on:
       - postgres
```

Account service also depends on `rabbitmq`, but maybe yuanfan has not yet gone that far. Anyway, in the future, it's gonna be dependent on `rabbitmq`.
```suggestion
      - postgres
      - rabbitmq
```